cobc/scanner.l

```diff
@@ -1377,19 +1465,20 @@ read_literal (const char mark, const char *type)
 	if (!i) {
 		cb_verify (cb_zero_length_lit, _("zero-length literal"));
 		cb_warning (COBC_WARN_FILLER,
-			    type[0] == 'N' ?
-			    _("national literal has zero length; a SPACE will be assumed") :
-			    _("alphanumeric literal has zero length; a SPACE will be assumed"));
+			    _("%s literal has zero length; a SPACE will be assumed"),
```

Comment:
We don't replace the two existing msgids by 1 shortly before the final release; this would need to be postponed after the release but also then only be done when changed for all of the similar ones to something like `"%s literal has zero length; %s will be assumed"`.
Please revert this part for now and feel free to keep it around in a separate PR.

Reply:
done